cmd/minikube/cmd/dashboard.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/docker/machine/libmachine"
+	"github.com/golang/glog"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+	"k8s.io/minikube/pkg/minikube/cluster"
+	"k8s.io/minikube/pkg/minikube/constants"
+)
+
+var (
+	noBrowser bool
+)
+
+// basically pull the new library with "go get"
+// onto your GOPATH
+// then everything should build and test
+// then you run "godep save ./..."
+// and it'll copy the package into vendor/
+
+// dashboardCmd represents the status command
+var dashboardCmd = &cobra.Command{
+	Use:   "open-dashboard",
+	Short: "Opens/displays the kubernetes dashboard URL for your local cluster",
+	Long:  `Opens/displays the kubernetes dashboard URL for your local cluster`,
+	Run: func(cmd *cobra.Command, args []string) {
+		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+		defer api.Close()
+		url, err := cluster.GetDashboardURL(api)
+		if err != nil {
+			glog.Errorln("Error getting machine status:", err)
+			os.Exit(1)
+		}
+		if noBrowser {
+			fmt.Fprintln(os.Stdout, url)
+		} else {
+			browser.OpenURL(url)
+		}
+	},
+}
+
+func init() {
+	dashboardCmd.Flags().BoolVarP(&noBrowser, "no-browser", "", false, "Flag to not display the dashboard in a browser but instead as a url that is output to the command line")
```

Comment:
Let's make this flag "--open-browser" and default it to true. That's easier than the double negative.